pygerduty/events.py

```diff
@@ -0,0 +1,78 @@
+# Events module for pygerduty version 2. These methods are compatible with
+# Pagerduty Events API.
+
+import urllib
+from v2 import (
+    Pagerduty,
+    IntegrationAPIError,
+    INTEGRATION_API_URL,
```

Comment:
Can you move the integration specific constant and expection to this module instead of importing from v2?